### PR TITLE
Add UT to DefaultPageMetaStore.commitFile

### PR DESCRIPTION
**Pull request by YichuanSun**
### What changes are proposed in this pull request?

Add unit test for DefaultPageMetaStore.commitFile() method, and rename some variables to more accurate names. 

### Why are the changes needed?

Improve the test coverage.

### Does this PR introduce any user facing changes?

No.

### CODE REVIEW

File: dora/core/client/fs/src/test/java/alluxio/client/file/cache/DefaultPageMetaStoreTest.java [hunk 1]
```diff
@@ -73,6 +73,15 @@ public void before() {
         MetricsSystem.METRIC_REGISTRY.getGauges().get(MetricKey.CLIENT_CACHE_PAGES.getName());
   }
 
+  @Test
+  public void commitFile() throws PageNotFoundException {
+    String newTempFile = "newTempFile";
+    PageId newTempPage = new PageId(newTempFile, 2L);
+    mMetaStore.addPage(mPage, mPageInfo);
+    mMetaStore.commitFile(mPage.getFileId(), newTempFile);
+    assertEquals(newTempFile, mMetaStore.getPageInfo(newTempPage).getPageId().getFileId());
```
Comment (jiacheliu3) [inline dora/core/client/fs/src/test/java/alluxio/client/file/cache/DefaultPageMetaStoreTest.java:82]: Are there more information you can assert on, other than just the file/page id? Are there anything like `file.isComplete`? Are there anythign like file length?

File: dora/core/client/fs/src/test/java/alluxio/client/file/cache/QuotaPagePageMetaStoreTest.java [hunk 2]
```diff
@@ -36,7 +36,7 @@
 /**
  * Tests for the {@link QuotaPageMetaStore} class.
  */
-public class QuotaPageMetaStoreTest extends DefaultMetaStoreTest {
+public class QuotaPagePageMetaStoreTest extends DefaultPageMetaStoreTest {
```
Comment (jiacheliu3) [inline dora/core/client/fs/src/test/java/alluxio/client/file/cache/QuotaPagePageMetaStoreTest.java:39]: nit: PagePage?

**Review by jiacheliu3 — APPROVED** [review]
mostly LGTM with one minor comment, thanks for the work!

File: dora/core/client/fs/src/test/java/alluxio/client/file/cache/QuotaPagePageMetaStoreTest.java (hunk 2 above)
Comment (YichuanSun) [reply dora/core/client/fs/src/test/java/alluxio/client/file/cache/QuotaPagePageMetaStoreTest.java]: Sorry about this..... have been fixed.

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**